routers/api/packages/rpm/rpm.go

```diff
@@ -33,11 +33,12 @@ func apiError(ctx *context.Context, status int, obj any) {
 
 // https://dnf.readthedocs.io/en/latest/conf_ref.html
 func GetRepositoryConfig(ctx *context.Context) {
+	distribution := strings.TrimSpace(ctx.Params("distribution"))
```

Comment:
I think you don't need the `strings.TrimSpace` because we don't trim any of the other parameters.